--- openwrt/include/kernel-defaults.mk
+++ openwrt1/include/kernel-defaults.mk
@@ -118,7 +118,8 @@
 		cp $(LINUX_DIR)/.config.set $(LINUX_DIR)/.config.prev; \
 	}
 	$(_SINGLE) [ -d $(LINUX_DIR)/user_headers ] || $(KERNEL_MAKE) INSTALL_HDR_PATH=$(LINUX_DIR)/user_headers headers_install
-	grep '=[ym]' $(LINUX_DIR)/.config.set | LC_ALL=C sort | $(MKHASH) md5 > $(LINUX_DIR)/.vermagic
+	# grep '=[ym]' $(LINUX_DIR)/.config.set | LC_ALL=C sort | $(MKHASH) md5 > $(LINUX_DIR)/.vermagic
+	cp $(TOPDIR)/vermagic $(LINUX_DIR)/.vermagic
 endef
 
 define Kernel/Configure/Initramfs
